src/components/transactions/Swap/SwapModalContent.tsx

```diff
@@ -152,6 +152,13 @@ export const SwapModalContent = ({
             collateral to perform this action
           </Trans>
         );
+      case ErrorType.FLASH_LOAN_NOT_AVAILABLE:
```

Comment:
I think the error condition needs to be added in this file as well
```
else if (shouldUseFlashloan && !collateralReserveData.flashLoanEnabled) {
    blockingError = ErrorType.FLASH_LOAN_NOT_AVAILABLE;
  }
 ```

Reply:
good catch, added